SConstruct

```diff
@@ -258,6 +258,7 @@ opts.Add(BoolVariable("scu_build", "Use single compilation unit build", False))
 opts.Add("scu_limit", "Max includes per SCU file when using scu_build (determines RAM use)", "0")
 opts.Add(BoolVariable("engine_update_check", "Enable engine update checks in the Project Manager", True))
 opts.Add(BoolVariable("steamapi", "Enable minimal SteamAPI integration for usage time tracking (editor only)", False))
+opts.Add("num_jobs", "Allow N jobs at once", "")
```

Comment:
```suggestion
opts.Add("num_jobs", "Use up to N jobs when compiling (equivalent to `-j N`). Defaults to max jobs - 1.", "")
```

And I'd move it further up, e.g. between `compiledb` and `verbose`. Nitpicky, but semantically it's more related to these early options than later ones like `steamapi` or `engine_update_check` which impact the actual code being compiled.

Reply:
Gotcha! I think I've been moving it down over and over as more vars have been added, haha. I'll fix that up asap.